public/app/plugins/panel/flamegraph/components/TopTable/FlameGraphTopTable.tsx

```diff
@@ -136,6 +137,14 @@ const FlameGraphTopTable = ({
     [rows, prepareRow, search, styles.matchedRow, styles.row, styles.cell, rowClicked]
   );
 
+  const handleScroll: React.UIEventHandler = (event) => {
+    const { scrollTop } = event.target as HTMLDivElement;
+
+    if (listRef.current !== null) {
+      listRef.current.scrollTo(scrollTop);
+    }
+  };
+
```

Comment:
Not really sure why this is necessary? Again, the changes in [this PR](https://github.com/grafana/grafana/pull/56555) took a lot into account in terms of scrolling in dashboards/explore - would have to revisit to make sure it's accounted for here :) 

Also must note that these changes will apply to the flamegraph in both the trace view and Phlare/Parca so we ideally have to test at least the trace view and Phlare in both dashboards and explore.